src/components/FilterPanel.jsx

```diff
@@ -50,15 +53,16 @@ export default function FilterPanel({ onClose }) {
     setMinAmount(query.minAmount);
     setGrades(query.grades);
     setMajors(query.majors);
+    setStates(query.states);
   }, [location]);
 
   const filters = {
     'What are you studying?': {
       comp: (
         <>
-          <AutocompleteFilter
+          <CustomAutocomplete
             freeSolo
-            value={majors}
+            value={majors || []}
```

Comment:
Why is this necessary? Also why not have the `|| []` logic in the component itself?

Reply:
because it was complaining about the following: `MUI: A component is changing the uncontrolled value state of Autocomplete to be controlled.`

Reply:
I tried doing it in the component itself but interestingly, it wouldn't let me type in the state filter. It's the same bug that we have in the scholarship form